search/indexing_api.py

```diff
@@ -254,6 +254,9 @@ def program_enrolled_user_mapping():
             'ancestors': NOT_ANALYZED_STRING_TYPE,
             'value': NOT_ANALYZED_STRING_TYPE,
             'order': LONG_TYPE
+        }},
+        'semester_enrollments': {'type': 'nested', 'properties': {
+            'semester': NOT_ANALYZED_STRING_TYPE
```

Comment:
Can you give me an example value for this field? Is the semester meant to be part of the program object?